.github/workflows/nightly.yml

```diff
@@ -233,6 +233,67 @@ jobs:
           AWS_SECRET_ACCESS_KEY: "${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}"
         run: make release-s3
 
+  release-cloudsmith:
+    runs-on: ubuntu-latest
+    needs:
+      - build-x86_64-unknown-linux-gnu-packages
+      - build-x86_64-unknown-linux-musl-packages
+      - build-aarch64-unknown-linux-musl-packages
+    steps:
+      - uses: actions/checkout@v1
+      - name: Push amd64 deb
+        id: push-deb-amd64
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: "push"
+          format: "deb"
+          owner: "timber"
+          repo: "vector-nightly"
+          distro: "any-distro"
+          release: "any-version"
+          republish: "true"
+          file: "target/artifacts/vector-amd64.deb"
+      - name: Push arm64 deb
+        id: push-deb-arm64
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: "push"
+          format: "deb"
+          owner: "timber"
+          repo: "vector-nightly"
+          distro: "any-distro"
+          release: "any-version"
+          republish: "true"
+          file: "target/artifacts/vector-arm64.deb"
+      - name: Push x86_64 RPM
+        id: push-rpm-x86_64
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: "push"
+          format: "rpm"
+          owner: "timber"
+          repo: "vector-nightly"
+          distro: "any-distro"
+          release: "any-version"
```

Comment:
We can't support centos7 after #3657 until https://github.com/rust-embedded/cross/issues/455 is resolved.

Reply:
Yeah - I am still playing with their package metadata. This will probably change before the final but I wanted to get some packages in and tested.